src/main/webapp/index.jsp

```diff
@@ -12,16 +12,16 @@
     <nav>
       <ul id="navigation">
         <li><a href="/">Home</a></li>
-        <li><a href="/feed.html">Public Feed</li>
+        <li><a href="/feed.html">Public Feed</a></li>
 
     <%
       if (isUserLoggedIn) {
         String username = (String) request.getAttribute("username");
     %>
-        <a href="/user-page.html?user=<%= username %>">Your Page</a>
-        <a href="/logout">Logout</a>
+        <li><a href="/user-page.html?user=<%= username %>">Your Page</a>
+        <li><a href="/logout">Logout</a>
     <% } else {   %>
-       <a href="/login">Login</a>
+       <li><a href="/login">Login</a>
```

Comment:
This is just fixing up the way that these links are viewed for consistency, because there was a weird space or something on one of them